lib/dash/dash_parser.js

```diff
@@ -497,6 +555,8 @@ shaka.dash.DashParser = class {
 
     // These steps are not done on manifest update.
     if (!this.manifest_) {
+      await this.periodCombiner_.combinePeriods(periods, context.dynamic);
```

Comment:
This was just run on line 542.  Why do it again now?

Reply:
The call above is the one I should have removed but left in this PR, sorry.